EESSI-pilot-install-software.sh

```diff
@@ -142,94 +142,79 @@ else
+    echo_green ">> EasyBuild seems to be working!"
+    $EB --version | grep "${REQ_EB_VERSION}"
+    if [[ $? -eq 0 ]]; then
+        echo_green "Found EasyBuild version ${REQ_EB_VERSION}, looking good!"
+    else
+        $EB --version
+        error "Expected to find EasyBuild version ${REQ_EB_VERSION}, giving up here..."
+    fi
     $EB --show-config
 else
     error "EasyBuild not working?!"
 fi
 
 echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALLPATH}..."
 
+# install GCC, using GCC easyblock with workaround for bug introduced in EasyBuild v4.3.1,
+# see https://github.com/easybuilders/easybuild-easyblocks/pull/2217
 export GCC_EC="GCC-9.3.0.eb"
 echo ">> Starting slow with ${GCC_EC}..."
-$EB ${GCC_EC} --robot
+$EB ${GCC_EC} --robot --include-easyblocks-from-pr 2217
 if [[ $? -eq 0 ]]; then
     echo_green "${GCC_EC} installed, yippy! Off to a good start..."
 else
     error "Installation of ${GCC_EC} failed!"
 fi
 
-# side-step to fix missing build dependency for Perl,
-# see https://github.com/easybuilders/easybuild-easyconfigs/pull/11200
-export PERL_EC="Perl-5.30.2-GCCcore-9.3.0.eb"
-echo ">> Taking a small side step to install ${PERL_EC}..."
-$EB --from-pr 11368 makeinfo-6.7-GCCcore-9.3.0.eb --robot && $EB --from-pr 11454 DB-18.1.32-GCCcore-9.3.0.eb --robot && $EB --from-pr 11200 --robot
-if [[ $? -eq 0 ]]; then
-    echo_green "${PERL_EC} installed via easyconfigs PR #11200, that was just a small side step, don't worry..."
-else
-    error "Installation of ${PERL_EC} failed!"
-fi
-
-# side-step to fix installation of CMake with zlib included in --filter-deps
-# see https://github.com/easybuilders/easybuild-easyblocks/pull/2187
-echo ">> Installing CMake with fixed easyblock..."
-$EB CMake-3.16.4-GCCcore-9.3.0.eb --include-easyblocks-from-pr 2187 --robot
-if [[ $? -eq 0 ]]; then
-    echo_green "CMake installation done, glad that worked out!"
-else
-    error "Installation of CMake failed, pfft..."
-fi
-
-# required to make sure that libraries like zlib that are listed in --filter-deps can be found by pkg-config
-# FIXME: fix this in EasyBuild framework!
-#        see https://github.com/easybuilders/easybuild-framework/pull/3451
-export PKG_CONFIG_PATH=$EPREFIX/usr/lib64/pkgconfig
-
-# FIXME custom installation of Qt5 with patch required to build with Gentoo's zlib
-# see https://github.com/easybuilders/easybuild-easyconfigs/pull/11385
-echo ">> Installing Qt5 with extra patch to use zlib provided by Gentoo..."
-$EB --from-pr 11385 --robot
+# install custom fontconfig that is aware of the compatibility layer's fonts directory
+# see https://github.com/EESSI/software-layer/pull/31
+export FONTCONFIG_EC="fontconfig-2.13.92-GCCcore-9.3.0.eb"
+echo ">> Installing custom fontconfig easyconfig (${FONTCONFIG_EC})..."
+cd ${TMPDIR}
+curl -OL https://raw.githubusercontent.com/EESSI/software-layer/master/easyconfigs/${FONTCONFIG_EC}
+cd - > /dev/null
+$EB ${FONTCONFIG_EC} --robot
```

Comment:
Shouldn't these be in reverse order?

Reply:
Ugh, yes, I actually meant to use `$TMPDIR/${FONTCONFIG_EC}` here... Will fix.